README.md

```diff
@@ -180,3 +180,5 @@ Support this project by becoming a sponsor. Your logo will show up here with a l
 ![https://opencollective.com/sous-chefs/sponsor/7/website](https://opencollective.com/sous-chefs/sponsor/7/avatar.svg?avatarHeight=100)
 ![https://opencollective.com/sous-chefs/sponsor/8/website](https://opencollective.com/sous-chefs/sponsor/8/avatar.svg?avatarHeight=100)
 ![https://opencollective.com/sous-chefs/sponsor/9/website](https://opencollective.com/sous-chefs/sponsor/9/avatar.svg?avatarHeight=100)
+
+[pyenv]: https://github.com/pyenv/pyenv
```

Comment:
```suggestion
[pyenv](https://github.com/pyenv/pyenv)
```

Reply:
FYI, this is not a typo.

This is setup as a reference point for https://github.com/sous-chefs/pyenv/pull/55/files#diff-04c6e90faac2675aa89e2176d2eec7d8R12 so applying the suggestion will create a link at the bottom of the README.

If you prefer to have the link next to the text, I can update it accordingly but will need to delete this line.